src/config/config.js

```diff
@@ -24,19 +24,19 @@ import enums from '../enums';
 export default {
   /**
    * @memberof module:config
-   * @property {Integer} preferHashAlgorithm Default hash algorithm {@link module:enums.hash}
+   * @property {Integer} preferredHashAlgorithm Default hash algorithm {@link module:enums.hash}
    */
-  preferHashAlgorithm: enums.hash.sha256,
+  preferredHashAlgorithm: enums.hash.sha256,
   /**
    * @memberof module:config
-   * @property {Integer} encryptionCipher Default encryption cipher {@link module:enums.symmetric}
+   * @property {Integer} preferredCipherAlgorithm Default encryption cipher {@link module:enums.symmetric}
    */
-  encryptionCipher: enums.symmetric.aes256,
+  preferredCipherAlgorithm: enums.symmetric.aes256,
```

Comment:
Sorry, last nitpick - probably this should be `preferredSymmetricAlgorithm`, for consistency with `preferredSymmetricAlgorithms`?

Reply:
I disagree because AEAD is also symmetric, I think the existing 'symmetric' stuff should be renamed instead, however they are not all internal, so perhaps it risks breaking too much?

Reply:
Well `preferredSymmetricAlgorithms` is named that way because of the name of the subpacket in the spec.. I don't think we can rename that. And - if you say "symmetric encryption algorithm", people think of AES, not EAX, no?

Reply:
Right, it's unfortunate they haven't updated the packet name (yet? :D) -- I don't find "symmetric encryption algorithm" especially descriptive, so not sure what other people would think :sweat_smile: I would still consider renaming the `enums` though, since also the spec do talk about "ciphers" here and there